stat/distuv/triangle.go

```diff
@@ -134,6 +135,66 @@ func (t Triangle) Rand() float64 {
 	return t.Quantile(rnd)
 }
 
+// Score returns the score function with respect to the parameters of the
+// distribution at the input location x. The score function is the derivative
+// of the log-likelihood at x with respect to the parameters
+//  (∂/∂θ) log(p(x;θ))
+// If deriv is non-nil, len(deriv) must equal the number of parameters otherwise
+// Score will panic, and the derivative is stored in-place into deriv. If deriv
+// is nil a new slice will be allocated and returned.
+//
+// The order is [∂LogProb / ∂Mu, ∂LogProb / ∂Sigma].
+//
+// For more information, see https://en.wikipedia.org/wiki/Score_%28statistics%29.
+func (t Triangle) Score(deriv []float64, x float64) []float64 {
```

Comment:
This is a new addition; it and its tests should be another PR (possibly the fix to `Laplace` too).

Reply:
If I don't add the Score methods, I can't test parameters and setParameters using the existing testDerivativeParam function.

Reply:
Mmm, OK. I think that changes the focus of the PR then, which is fine. It would become

```
stat/distuv: add Score method to Triangle and Uniform

Also tests and fix Laplace Score.
```

Reply:
Done.